docs/reference/api/docker_remote_api.md

```diff
@@ -125,6 +125,7 @@ This section lists each version from latest to oldest.  Each listing includes a
 * `GET /info` now returns `KernelMemory` field, showing if "kernel memory limit" is supported.
 * `POST /containers/create` now takes `PidsLimit` field, if the kernel is >= 4.3 and the pids cgroup is supported.
 * `GET /containers/(id or name)/stats` now returns `pids_stats`, if the kernel is >= 4.3 and the pids cgroup is supported.
+* `POST /containers/create` now takes Storage Options `--storage-opt`. e.g `--storage-opt size:120G` can set the container rootfs size to 120G at creation time. 
```

Comment:
Oh, sorry, :cry: this is describing the API, so I don't think we need to / should mention the `--storage-opt` flag here; only the `StorageOpt`